portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,10 +82,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   private String getParameter(HttpServletRequest request, String name, String defaultValue) {
     String value = request.getParameter(name);
-    if (value == null) {
-      return defaultValue;
-    }
-    return value;
+   return request.getParameter(name) == null ? defaultValue : value;
```

Comment:
Whoops, since you're keeping the value field, we can do
`return value == null? defaultValue : value`